homeassistant/components/jvc_projector/config_flow.py

```diff
@@ -103,7 +103,11 @@ async def async_step_reauth_confirm(
             else:
                 self.hass.config_entries.async_update_entry(
                     self._reauth_entry,
-                    data={CONF_HOST: host, CONF_PORT: port, CONF_PASSWORD: password},
+                    data={
+                        CONF_HOST: host,
+                        CONF_PORT: port,
+                        CONF_PASSWORD: password,
+                    },
```

Comment:
Please revert

Reply:
@joostlek sorry do not understand what I have to do, what you mean with revert? there should be no change here.